dotcom-rendering/src/web/components/NewslettersBadge.tsx

```diff
@@ -0,0 +1,65 @@
+import { css } from '@emotion/react';
+import { brandAlt, headline, neutral } from '@guardian/source-foundations';
+
+const textStyle = (textColour: string) => css`
+	${headline.small({ fontWeight: 'bold' })};
```

Comment:
```suggestion
	${headline.xxsmall({ fontWeight: 'bold' })};
```
`xxsmall` is 20px

See: https://guardian.github.io/source/?path=/story/packages-source-foundations-typography--page#headline